rosie/chamber_of_deputies/tests/test_election_expenses_classifier.py

```diff
@@ -11,16 +11,16 @@ class TestElectionExpensesClassifier(TestCase):
     def setUp(self):
         self.dataset = pd.read_csv('rosie/chamber_of_deputies/tests/fixtures/election_expenses_classifier.csv',
                                    dtype={'name': np.str, 'legal_entity': np.str})
-        self.subject = ElectionExpensesClassifier()
+        self.election_expenser_classifier = ElectionExpensesClassifier()
```

Comment:
Just wondering: as each test file refers to only one classifier I wouldn't bother shortening it to `self.classifier`…

Reply:
I aiming to achieve people that have no context of a classifier, to understand what is this referencing to.

How do you `self.classifier` helping with it? Or is this tip addressing something else that I didn't get here?

Reply:
> I aiming to achieve people that have no context of a classifier

Hopefully readers know which file they are reading… or maybe I'm just too optimistic.

But nothing against `self.election_expenser_classifier` _per se_ except that I was trying something a bit shorter (minor enhancement in code readability) and as meaningful as before (guessing the file name would be a context to inform the reader about which classifier we're talking about). But that was just a minor suggestion… feel free to throw it away ; )

Reply:
In that way, I'm with @cuducos, I prefer something meaningful but simple at the same way :)